drivers/starburst/src/metabase/driver/implementation/driver_helpers.clj

```diff
@@ -33,7 +33,7 @@
                               :native-parameters               true
                               :expression-aggregations         true
                               :binning                         true
-                              :foreign-keys                    true
+                              :foreign-keys                    false
```

Comment:
Is there a doc for this for my reference?

Reply:
No Metabase doc. I realized this when I found that all the tests checking foreign keys failed. And it turns out that, when using the Trino JDBC driver, `DatabaseMetaData.getImportedKeys()` doesn't appear to work.

Reply:
I meant do that says Trino does not support Foreign Keys. I just can't find it in the official docs.

Reply:
One thing I'm researching in parallel is how come Presto Metabase driver supports foreign keys. It may be possible to have the Starburst driver support foreign keys, but it appears it never had.